deploy-as-code/helm/charts/core-services/egov-mdms-service/values.yaml

```diff
@@ -13,7 +13,7 @@ ingress:
 initContainers:
   gitSync:
     enabled: true
-    repo: "git@github.com:egovernments/egov-mdms-data"
+    repo: "git@github.com:misdwss/mdms-mgramseva"
```

Comment:
Don't modify helm chart for this. You can override this property in the env file like [this](https://github.com/egovernments/DIGIT-DevOps/blob/digit-works/deploy-as-code/helm/environments/works-dev.yaml#L189).

Reply:
Changed.